.gitlab/build_lassen.yml

```diff
@@ -14,13 +14,9 @@
     - if: '$CI_COMMIT_BRANCH =~ /_lnone/ || $ON_LASSEN == "OFF"' #run except if ...
       when: never
     - when: on_success
-
-####
-# Load required CUDA module
-.with_cuda:
   before_script:
     - module load cuda/11.2.0
-    - module load cmake/3.21.1
+    - module load cmake/3.18.0
```

Comment:
FYI. RAJA and Umpire require cmake min version 3.20.

Reply:
I was having some serious issues with an intermittent CMake issue that was only in CI. I wanted to limit the amount of variables so i picked 3.18.0 across the board (minus Tioga which doesn't have that version).

We aren't actually requiring this version at all and this ended up not being the issue.